app/frontend/src/pages/chat/Chat.tsx

```
@@ -294,6 +302,26 @@ const Chat = () => {
             case "includeCategory":
                 setIncludeCategory(value);
                 break;
+            case "sortBy":
+                const newSortBy = value as string;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)